### PR TITLE
Scripts i18n proof-of-concept

### PR DESCRIPTION
**Pull request by plata**
`resources` should be filled with the merged content of the language `.json` files which have to be loaded from every used repository. To do this, we need a `RepositoryDTO` with a list of `CategoryDTO` and a `Map` for the translations (language, json String).

### CODE REVIEW

File: phoenicis-scripts/src/main/java/org/phoenicis/scripts/nashorn/NashornScriptInterpreter.java [hunk 1]
```diff
@@ -32,7 +33,30 @@ public NashornScriptInterpreter(NashornEngineFactory nashornEngineFactory) {
 
     @Override
     public void runScript(String scriptContent, Consumer<Exception> errorCallback) {
-        nashornEngineFactory.createEngine().eval(scriptContent, errorCallback);
+        final String language = Locale.getDefault().getLanguage();
+        final String i18nextInit = "i18next.init({                                                                     "
+                + String.format("  lng: '%s',                                                             ", language)
```
Comment (madoar) [inline phoenicis-scripts/src/main/java/org/phoenicis/scripts/nashorn/NashornScriptInterpreter.java]: I don't like to define code as a String.
This looks quite ugly and hard to maintain.
Is there another way to set this?

Comment (plata) [inline phoenicis-scripts/src/main/java/org/phoenicis/scripts/nashorn/NashornScriptInterpreter.java]: I don't think so (I have to call JS).

**Comment by qparis** [comment]
I still don't understand why we are not using Java tr here

**Comment by madoar** [comment]
@plata is this PR still required?

**Comment by plata** [comment]
No.

**Pull request closed without merging**